.travis.yml

```diff
@@ -204,6 +204,12 @@ script:
           echo -e '\052\052 FAILED -- MAKE DOC-NITS';
           travis_terminate 1;
       fi
+    - if test -n "$MARKDOWNLINT" ; then
+          echo -e "====START MARKDOWNLINT====";
+          gem install mdl || travis_terminate 1;
+          mdl -s util/style.rb README.md || travis_terminate 1;
```

Comment:
You run the lint only on README.md here.

Reply:
fixed.